scripts/docker/requirements-20.04.txt

```diff
@@ -19,3 +19,5 @@ python3-dev=3.8.2-0ubuntu2
 python3.8-venv
 cargo
 ninja-build
+patchelf
```

Comment:
I think we are removing this in a separate PR?
https://github.com/tenstorrent/tt-metal/issues/11782

Reply:
Yes, we are https://github.com/tenstorrent/tt-metal/pull/11884